.gitignore

```diff
@@ -53,3 +53,4 @@ src/mixxx.rc.include
 src/mixxx.res
 src/test/**/*.actual
 res/qrc_mixxx.cc
+/.vs
```

Comment:
We do not add IDE related files in .gitignore. 
I suggest to use a global .gitignore for that: https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
Done ! Thanks for the tip